### ISSUE TITLE
Secondary option and line width

### ISSUE THREAD

**Issue by KlutzyBubbles** [shedaniel/LightOverlay-Fabric#130]
I may be mis understanding, but I cannot for the life of me figure out what the secondary light level does. At first I thought it was just another level to change the color of the X's, but it seems to only make the existing X's dissapear.

I then read something about blue cross for spawners (i dont know what that means), i tried placing a spawner to see if any blue X's would appear but alas... nothing.

As for line width, its another option i cannot figure out what it does, whether it is set at 1, 2, 7 etc the lines look the same. is it meant to thicken the lines visually? if so that isnt happening for me.

tl;dr Secondary Light Level, Line Width and Secondary Color options don't seem to do anything

**Comment by SedatedJdawg** [comment]
I'm curious about this too!

**Comment by dbrighthd** [comment]
line width is still not doing anything for me
